api/db/data.js

```diff
@@ -0,0 +1,26 @@
+const validate = require("../validate");
+const { ParameterizedQuery: PQ } = require("pg-promise");
+
+const db = require("./connection.js");
+
+async function querySuggestions(id) {
+  await validate.checkHandle(id);
+
+  // Create a parameterized query with the id
+  const query = new PQ({ text: "SELECT * FROM oapen_suggestions.suggestions WHERE item_id = $1", values: [id] });
```

Comment:
Might want to break this out into a library function

Reply:
The query? How?

Reply:
absolutely not a big deal at all, so low prio, but I usually keep routes and library/helper functions seperate, could make a `getSuggestionsByBookID(id)` that could be used in a few places. again not a big deal at all, a small nitpick haha

Reply:
I'm not familiar with JS libraries - does ParameterizedQuery handle parameter sanitization?

Reply:
It should handle SQL injection https://github.com/brianc/node-postgres/wiki/FAQ#8-does-node-postgres-handle-sql-injection
And we will code out the validate.checkHandle(id) function to further validate the input